### DISCUSSION TITLE
Bottom AND side description

### DISCUSSION THREAD

**Discussion by tacman** [biati-digital/glightbox#521] [Ideas]
I'd like to have a bottom description and a side (left or right) description.

I'd use the bottom description for a title, artist, year, copyright and some other short but important information.  Then I'd use the side area for all the other details about the image.  

If it's possible, I think it'd be a cool feature.

**Comment by gingerchew** [comment]
Hey there @tacman

I have had to convert a couple of your issues into discussions already, so figure I'd let you know why. Because GLightbox is a 2 person operation, myself and the maintainer, we limit issues to only being for bugs. If you have a question or an idea, we really encourage you to create those as discussions using the Q&A or Ideas tags respectively.

I'm going to convert this to an Ideas discussion.

Thank you for your interest in GLightbox :)

**Comment by tacman** [comment]
> I'm going to convert this to an Ideas discussion.

Understood!  Perhaps get rid of the "Feature Request" option in Issues, so others don't do the same.
